client/src/theme/styles/components/frontend/project-collection/_header.scss

```diff
@@ -78,22 +66,23 @@ $_font-family: 'sofia-pro', 'trueno', sans-serif;
   &__title {
     margin: 0;
     font-family: $_font-family;
-    font-size: 22px;
+    font-size: 26px;
     font-weight: 500;
-    hyphens: none;
     line-height: 1.32;
+    color: #363636;
```

Comment:
There should be a SCSS or CSS variable for this color.

Reply:
Please confirm for any other color values that you're hard-coding currently.

Reply:
I found two hard coded colors in the _variables.scss file and will be sure to use those color variables from here on out.